### DISCUSSION TITLE
Implement a method to control how often the extended script runs

### DISCUSSION THREAD

**Discussion by MadsMighty** [RandomNinjaAtk/docker-lidarr-extended#27] [Ideas]
With the current script running every 15 minutes, it's hammering my CPU constantly since it starts scanning again as soon as the first round is done.

Can this be added as a variable for the docker template? Or am I maybe missing something really obvious on where to set the script delay/timing.

**Comment by RandomNinjaAtk** [comment]
Read the readme, it’s already there as an option

**Reply by MadsMighty** [reply]
Terribly sorry about that, my template didn't pull down the updates you made. I'll make the changes on my end
